src/py42/services/detectionlists/departing_employee.py

```diff
@@ -178,14 +184,15 @@ def update_departure_date(self, user_id, departure_date):
 
         Args:
             user_id (str): The Code42 userUid of the user.
-            departure_date (date): Date in yyyy-MM-dd format. Date is treated as UTC.
+            departure_date (str or datetime instance): Date in yyyy-MM-dd format. Date is treated as UTC.
```

Comment:
Don't need instance in the arg type part.  Should be `(str or datetime)`.  In the description, though, mention it can also be an instance of a `datetime` object.

Reply:
Changed.